crates/holochain_persistence_lmdb/src/cas/lmdb.rs

```diff
@@ -33,55 +31,109 @@ impl Debug for LmdbStorage {
 }
 
 impl LmdbStorage {
-    pub fn new<P: AsRef<Path> + Clone>(
-        db_path: P,
-        initial_map_bytes: Option<usize>,
-    ) -> LmdbStorage {
-        LmdbStorage {
+    pub fn new<P: AsRef<Path> + Clone>(db_path: P, initial_map_bytes: Option<usize>) -> Self {
+        Self::wrap(&LmdbInstance::new(CAS_BUCKET, db_path, initial_map_bytes))
+    }
+
+    pub fn wrap(lmdb: &LmdbInstance) -> Self {
```

Comment:
```suggestion
    pub fn wrap(lmdb: LmdbInstance) -> Self {
```

It is a code smell to take something by reference and immediately clone it. Let your caller decide if they want to clone it or not.